qt/widgets/common/src/AlternateCSPythonLexer.cpp

```diff
@@ -82,3 +82,27 @@ QFont AlternateCSPythonLexer::defaultFont(int style) const {
   Q_UNUSED(style);
   return m_font;
 }
+
+///**
+// * Returns the keywords used by the lexer for a given set.
+// * @param set The keyword set to retrieve (1 is for keywords)
+// * @return A string containing the keywords
+// */
+const char *AlternateCSPythonLexer::keywords(int set) const {
+  if (set == 1) {
+    // Retrieve default Python keywords from qscintilla
+    const char *defaultKeywords = QsciLexerPython::keywords(1);
+    QStringList keywordList = QString::fromUtf8(defaultKeywords).split(" ");
+
+    for (const QString &keyword : m_customKeywords) {
+      if (!keywordList.contains(keyword)) {
+        keywordList.append(keyword);
+      }
+    }
+
+    QByteArray m_keywordsBuffer = keywordList.join(" ").toUtf8();
+    return m_keywordsBuffer.constData();
+  }
```

Comment:
I think you could simplify this (since it will be called a lot).

Simply add the `defaultKeywords` and `" True False"` to a string stream called `ss` (to concatenate them).
Then return `ss.str().c_str()`

I don't think that you need to spend the time changing types just to check if the keywords already exist, I don't think it will matter if they do.
You could also just define the custom keywords in this function, since it is not used anywhere else in the class.